hirte.spec.in

```diff
@@ -149,17 +149,38 @@ This package contains the service controller command line tool.
 %{_mandir}/man1/hirtectl.*
 
 
+%package python3-pyhirte
+Summary: Python bindings for Hirte
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+Requires:       python3.9dist(dasbus)
```

Comment:
Why do we need to include python3.9 version of dasbus? Shouldn't we use the name without specific python version?

```
Requires: python3-dasbus
```

Reply:
Ah, you are right. Changed it.